src/server/coprocessor/executor/scanner.rs

```diff
@@ -186,6 +196,48 @@ pub mod test {
         }
     }
 
+    pub fn prepare_index_data(key_number: usize, table_id: i64, index_id: i64) -> Data {
```

Comment:
I think `prepare_index_data` and `prepare_table_data` should be moved to `tableScan` and `indexScan`.
Because the scanner should not know anything about table or index.

To test the scanner itself, we can just create some arbitrary keys, it doesn't have to be in the table format.